src/components/getting-started.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react"
+
+import { Layout } from "../components/layout"
+import SEO from "../components/seo"
+import styled from "styled-components"
+import colors from "../common/colors"
+import { SlantedBorderElement } from "../common/slanted-borders-helpers.js"
+
+import breakpoints from "../common/breakpoints.js"
+
+const MainContainer = styled(SlantedBorderElement)`
+  color: ${colors.typography.light};
+  background-color: ${colors.backgrounds.dark1};
+`
+const Title = styled.h2`
+  padding-top: 32px;
+  text-align: center;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 47px;
+
+  @media (min-width: ${breakpoints.mobile}px) {
+    font-size: 36px;
+  }
+`
+const Section = styled.div`
+  display: flex;
+  min-height: 200px;
+  justify-content: center;
+  align-items: center;
+  min-width: 100px;
+  transform: translateX(5%);
+  @media (min-width: ${breakpoints.mobile}px) {
+    min-width: 150px;
+  }
+  @media (min-width: ${breakpoints.tablet}px) {
+    transform: translateX(-15%);
+    min-width: 200px;
+  }
+`
+const RightAlignSection = styled(Section)`
+  @media (min-width: ${breakpoints.mobile}px) {
+    min-width: 150px;
+  }
+  @media (min-width: ${breakpoints.tablet}px) {
+    transform: translateX(15%);
+    min-width: 300px;
+  }
+`
+const BigNumber = styled.div`
+  font-size: 60px;
+  align-items: center;
+  justify-content: flex-start;
+  width: 50px;
+  font-weight: 600;
+  font-size: 48px;
+  text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+
+  @media (min-width: ${breakpoints.mobile}px) {
+    font-size: 144px;
+    width: 100px;
+  }
+`
+const Description = styled.div`
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  width: 300px;
+  @media (min-width: ${breakpoints.mobile}px) {
+    min-width: 150px;
+  }
+  @media (min-width: ${breakpoints.tablet}px) {
+    transform: translateX(15%);
```

Comment:
Margin or padding is preferred for adding space between elements.

Reply:
Any updates on this?

Reply:
FIxed, I believe